### PR TITLE
[HOTFIX] Add livy build in .travis

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?

livy integration test is ignored, this PR add livy integration test in travis

### What type of PR is it?
[Hot Fix]

### Todos
* [ ] - Task

### What is the Jira issue?
* No jira created

### How should this be tested?
CI pass

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Isn't

`# Test interpreter modules`

and 

`# Test spark module for 2.1.0 with scala 2.11, livy`

test livy?

**Comment by zjffdu** [comment]
livy has integration test, so it require `verify` instead `test`.  That's why I add them into the the build with `verify` phase.

File: .travis.yml [hunk 1]
```diff
@@ -46,12 +46,14 @@ matrix:
...
 
     # Test selenium with spark module for 1.6.3
-    - jdk: "oraclejdk7"
-      env: TEST_SELENIUM="true" SCALA_VER="2.10" SPARK_VER="1.6.3" HADOOP_VER="2.6" PROFILE="-Pspark-1.6 -Phadoop-2.6 -Phelium-dev -Pexamples" BUILD_FLAG="package -DskipTests -DskipRat" TEST_FLAG="verify -DskipRat" TEST_PROJECTS="-pl .,zeppelin-interpreter,zeppelin-zengine,zeppelin-server,zeppelin-display,spark-dependencies,spark -Dtest=org.apache.zeppelin.AbstractFunctionalSuite -DfailIfNoTests=false"
+    - sudo: required
+      jdk: "oraclejdk7"
+      env: TEST_SELENIUM="true" SCALA_VER="2.10" SPARK_VER="1.6.3" HADOOP_VER="2.6" LIVY_VER="0.2.0" PROFILE="-Pspark-1.6 -Phadoop-2.6 -Phelium-dev -Pexamples -Plivy-0.2" BUILD_FLAG="package -DskipTests -DskipRat" TEST_FLAG="verify -DskipRat" TEST_PROJECTS="-pl .,zeppelin-interpreter,zeppelin-zengine,zeppelin-server,zeppelin-display,spark-dependencies,spark,livy -Dtest=org.apache.zeppelin.AbstractFunctionalSuite -DfailIfNoTests=false"
```
Comment (felixcheung) [inline .travis.yml]: does livy 0.2 need to go with selenium tests? seems like selenium tests can be flaky

**Comment by zjffdu** [comment]
@Leemoonsoo @felixcheung Please help review. I add livy integrate test with python2 and python3 test. Here's the travis link https://travis-ci.org/zjffdu/zeppelin/builds/226240044

**Pull request closed without merging**